RFS/src/main/java/org/opensearch/migrations/snapshot/creation/tracing/CreateSnapshotContext.java

```diff
@@ -0,0 +1,61 @@
+package org.opensearch.migrations.snapshot.creation.tracing;
```

Comment:
This extra layer `snapshot.creation` doesn't make much sense to me.  If this package names are weird, its because we are working backward from functionality in an executable in a shard library space rather treating this as a shared library with a clear sense of what context is required at each level.

I'd rather have generic IContext use in the library and concrete implementation in the executable project so we can keep this clearer.  What do you think?

Reply:
I plan on moving everything in this directory to the executable project.  I'm doing the class refactorings now, but holding off on the file moves because additional files will need to move around rather than just the ones that I've created here.  I can do that as a fast-follow, but I'd like to do it in such a way to minimize conflicts.

As per the package name - "snapshot.creation" didn't sit completely at ease with me.  I wasn't sure what this package should be called longer term (it definitely shouldn't be 'com.rfs', nor should it likely go under the top-level "org.opensearch.migrations" - I'm open to other suggestions.